eclass/toolchain.eclass

```diff
@@ -129,46 +128,36 @@ LICENSE="GPL-3+ LGPL-3+ || ( GPL-3+ libgcc libstdc++ gcc-runtime-library-excepti
 IUSE="test vanilla +nls"
 RESTRICT="!test? ( test )"
 
-tc_supports_dostrip() {
-	case ${EAPI} in
-		5|6) return 1 ;;
-		7|8) return 0 ;;
-		*) die "Update apply_patches() for ${EAPI}." ;;
-	esac
-}
-
-tc_supports_dostrip || RESTRICT+=" strip" # cross-compilers need controlled stripping
-
 TC_FEATURES=()
 
 tc_has_feature() {
 	has "$1" "${TC_FEATURES[@]}"
 }
 
 if [[ ${PN} != "kgcc64" && ${PN} != gcc-* ]] ; then
-	IUSE+=" debug +cxx +nptl" TC_FEATURES+=(nptl)
+	IUSE+=" debug +cxx +nptl" TC_FEATURES+=( nptl )
```

Comment:
I'd rather not clutter the history with purely cosmetic whitespace changes (here and all following instances).

But YMMV, not a blocker for this patch.